functorch/csrc/BatchRulesViews.cpp

```diff
@@ -152,7 +153,10 @@ std::tuple<Tensor,optional<int64_t>> _unsafe_view_batch_rule(
 }
 
 Tensor trace_decomp(const Tensor& self) {
```

Comment:
can we make this a macro? or perhaps a boxed fallback?

Reply:
Done... depends on https://github.com/pytorch/pytorch/pull/76493